app/ServerProviders/AWS.php

```
@@ -21,6 +23,194 @@ public static function id(): string
+                return [$vpcId, is_string($ip) && $ip !== '' ? $ip : null];
+            }
+        }
+
+        $vpcId = $instance['VpcId'] ?? null;
+        $ip = $instance['PrivateIpAddress'] ?? null;
+
+        return [
+            is_string($vpcId) && $vpcId !== '' ? $vpcId : null,
+            is_string($ip) && $ip !== '' ? $ip : null,
+        ];
+    }
```

PHPMD (2.15.0)

[warning] The method placementOf() has a Cyclomatic Complexity of 10. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)